BrushedMotorControllerFirmware/boards/nrf52840dongle_nrf52840.overlay

```diff
@@ -2,59 +2,67 @@
 	status="disabled";
 };
 
+//Hardware v 1.X
```

Comment:
```suggestion
```

Reply:
It's information printed on the board itself, I want to keep it in the overlay not to mix up overlays and hardwares